app/grants/management/commands/grants_cv.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+"""
+
+Copyright (C) 2021 Gitcoin Core
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published
+by the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""
+
+from pprint import pprint
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+import requests
+from grants.models import Grant
+
+
+class Command(BaseCommand):
+
+    help = 'pulls data from https://twitter.com/austingriffith/status/1529825114597908484 and inserts into db'
+
+    def handle(self, *args, **options):
+
+        def run_query(q, url):
+            request = requests.post(url,
+                                    '',
+                                    json={'query': q})
+            if request.status_code == 200:
+                return request.json()
+            else:
+                raise Exception('Query failed. return code is {}.      {}'.format(request.status_code, query))
+
+        queryGrants = """
+        query getGrants {
+          grants(orderBy: id, orderDirection: asc, first: 100) {
+            id
+            votes {
+              id
+              amount
+              createdAt
+            }
+            releases {
+              id
+              amount
+              createdAt
+            }
+          }
+        }
+        """
+
+        urls = []
+        if not settings.DEBUG or True:
+            urls += ['https://api.studio.thegraph.com/query/20308/gtc-conviction-voting-mainnet/v0.0.1']
```

Comment:
@DanieleSalatti are these the final prod subgraphs?

i updated below to allow it to sum the mainnet/optimism stake amounts per @austingriffith instruction in tg

Reply:
waiting on this sign off and then we can get this rolled out 🙌🏼

Reply:
ok confirmed - these are the final prod subgraphs